src/parser/Parser.ts

```diff
@@ -933,7 +933,8 @@ export class Parser {
                     });
                 }
 
-                if (match(Lexeme.Else)) {
+                if (previous().kind !== Lexeme.Newline && match(Lexeme.Else)) {
+                    console.log("Else 2");
```

Comment:
Looks like you forgot to remove this `console.log` statement.

Reply:
indeed, will fix it.